packages/manager/apps/web/client/app/private-database/whitelist/private-database-whitelist.html

```diff
@@ -10,82 +10,98 @@
-                    ></i>
-                </oui-datagrid-column>
-                <oui-datagrid-column
-                    data-title="'privateDatabase_tabs_whitelist_list_sftp' | translate"
-                    data-property="sftp"
-                >
-                    <i
-                        class="oui-icon oui-icon-success"
-                        data-ng-show="$row.sftp"
-                        aria-hidden="true"
-                    ></i>
-                </oui-datagrid-column>
-                <oui-datagrid-column
-                    data-title="'privateDatabase_tabs_whitelist_list_status' | translate"
-                    data-property="name"
+            <div class="mt-5">
+                <h4
+                    data-translate="privateDatabase_add_authorized_ip_title"
+                ></h4>
+                <button
+                    class="oui-button oui-button_secondary mb-3 mt-5 mt-lg-0"
+                    type="button"
+                    data-translate="privateDatabase_tabs_whitelist_list_add_whitelist"
+                    data-ng-click="listCtrl.createWhitelist()"
+                ></button>
+
+                <oui-datagrid
+                    data-rows="listCtrl.whitelistIps"
+                    data-row-loader="listCtrl.transformItem($row)"
                 >
-                    <span
-                        class="label label-info"
-                        data-ng-if="$row.creating"
-                        data-translate="privateDatabase_tabs_whitelist_list_status_creating"
-                    ></span>
-                    <span
-                        class="label label-info"
-                        data-ng-if="$row.updating"
-                        data-translate="privateDatabase_tabs_whitelist_list_status_updating"
-                    ></span>
-                    <span
-                        class="label label-info"
-                        data-ng-if="$row.deleting"
-                        data-translate="privateDatabase_tabs_whitelist_list_status_deleting"
-                    ></span>
-                    {{ $row.name | wucSliceContent: 50 }}
-                </oui-datagrid-column>
-                <oui-action-menu data-compact data-placement="end">
-                    <oui-action-menu-item
-                        data-on-click="listCtrl.editWhitelist($row)"
-                        ><span
-                            data-translate="privateDatabase_tabs_whitelist_list_popover_edit"
+                    <oui-datagrid-column
+                        data-title="'privateDatabase_tabs_whitelist_list_ip_network' | translate"
+                        data-property="ip"
+                        data-sortable
+                    ></oui-datagrid-column>
+                    <oui-datagrid-column
+                        data-title="'privateDatabase_tabs_whitelist_list_service' | translate"
+                        data-property="service"
+                    >
+                        <i
```

Comment:
```suggestion
                        <span
```

Reply:
Seen with @marie-j , code was only moved. We will leave this for now.